hedera-node/config.txt

```diff
@@ -24,9 +24,9 @@ app,		HederaNode.jar
 # ** END REMOVE FROM SDK RELEASES **
 
  address,  A, Alice,    1, 127.0.0.1, 50204, 127.0.0.1, 50204, 0.0.3
- address,  B, Bob,      1, 127.0.0.1, 50205, 127.0.0.1, 50205, 0.0.4
+#address,  B, Bob,      1, 127.0.0.1, 50205, 127.0.0.1, 50205, 0.0.4
```

Comment:
nit : we can still use a 3 node network locally

Reply:
Reverted as in master